gdal/src/test/scala/geotrellis/contrib/vlm/gdal/GDALRasterSourceSpec.scala

```diff
@@ -109,8 +109,11 @@ class GDALRasterSourceSpec extends FunSpec with RasterMatchers with BetterRaster
       GDALRasterSource(GDALDataPath(p)).extent should be (GeoTiffRasterSource(p).extent)
     }
 
-    it("should fail on creation of the GDALRasterSource on a malformed URI") {
-      an[MalformedURLException] should be thrownBy GDALRasterSource(GDALDataPath("file:/random/path/here/N49W155.hgt.gz"))
+    ignore("should fail on creation of the GDALRasterSource on a malformed URI") {
+      //an[MalformedURLException] should be thrownBy GDALRasterSource(GDALDataPath("file:/random/path/here/N49W155.hgt.gz"))
+      val result = GDALRasterSource(GDALDataPath("file:/random/path/here/N49W155.hgt.gz"))
+      println(s"\n\nThis is the result: ${result.vsiPath}")
+      println(s"This is the result: ${Schemes.isVSIFormatted(result.vsiPath)}")
```

Comment:
Why `ignore`? The path doesn't exist but it reads as valid, I think it should parse well?

Reply:
It was copied from Jakes tests, dunno

Reply:
@echeipesh there is no scheme in this uri; parser wont figure out that file:/ is a typo and it should be file://